public/video-ui/src/components/ReactApp.js

```diff
@@ -37,6 +37,18 @@ class ReactApp extends React.Component {
     this.props.appActions.updateSearchTerm(searchTerm);
   };
 
+  getEditableFields = () => {
+    const allFields = this.props.checkedFormFields;
+    if (!allFields) {
+      return [];
+    }
```

Comment:
We don't actually so I'll remove it. I thought it might be undefined when the page loads but it is not.